addons/mail/static/src/new/rtc/call_participant_card.js

```diff
@@ -32,7 +32,7 @@ export class CallParticipantCard extends Component {
     get showConnectionState() {
         return Boolean(
             this.isOfActiveCall &&
-                !(this.props.session.channelMember?.persona.partner?.id === this.user.partnerId) &&
+                !(this.props.session.channelMember?.persona.id === this.user.partnerId) &&
```

Comment:
Won't this possibly conflict if the persona is for a guest that has the same id as current user?

Reply:
Yes this can happen. I'm ok to keep this problem for now as long as we do comparison of `id`.
I don't want to solve this issue now by increasing the amount of code (compare `type` and `id`).

The correct solution is to compare records/models. Sometimes the data are not records (`seenInfos`), and sometimes the code was using non-record for comparison (`this.user.partnerId`, `user` being the `user` service).

I will do follow-up PR to clean the code further, which should automatically fixes this class of problems (conflicting ids of guests and partners)